web/src/input_pill.ts

```diff
@@ -25,10 +25,15 @@ type InputPillCreateOptions<T> = {
     $container: JQuery;
     pill_config?: {
         show_user_status_emoji?: boolean;
+        allow_accessible_users_only?: boolean;
     };
     create_item_from_text: (
         text: string,
         existing_items: InputPillItem<T>[],
+        pill_config?: {
+            show_user_status_emoji?: boolean;
+            allow_accessible_users_only?: boolean;
+        },
```

Comment:
Should we define an `InputPillConfig` type that can be referenced by each of these items, rather than copying the list of properties in each place?